config/locales/en.yml

```diff
@@ -21,6 +21,9 @@ en:
       spree/order:
         payment_state: Payment State
         shipment_state: Shipment State
+    errors:
+      messages:
+        taken: "There's already an account for this email. Please login or reset your password."
```

Comment:
looks like this would override the generic message refering to the email. I believe we're missing the model and field this should be applied to. A quick google search returns me https://stackoverflow.com/a/18578028